views/cypress/tests/item.spec.js

```diff
@@ -150,4 +152,43 @@ describe('Items', () => {
             );
         });
     });
+
+    describe('Import', () => {
```

Comment:
I think better to move import and then export in separate `spec.js` file. Because it take more time and previous tests when fail can impact this test